src/main/scala/firrtl/passes/memlib/MemConf.scala

```diff
@@ -22,7 +22,7 @@ object MemPort {
     s.split(",").toSeq.map(MemPort.apply).map(_ match {
       case Some(x) => x
       case _ => throw new Exception(s"Error parsing MemPort string : ${s}")
-    }).groupBy(identity).mapValues(_.size)
+    }).groupBy(identity).mapValues(_.size).toMap
```

Comment:
should use `groupMap` here in 2.13.

Reply:
Is that backwards compatible? Please make a follow up PR.